pkgs/development/libraries/sfml/sfml_git.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, cmake, mesa, libX11, freetype, libjpeg, openal, libsndfile
+, glew, libXrandr, libXrender
+}:
+stdenv.mkDerivation rec {
+  name = "sfml-git-20110428";
```

Comment:
It's not possible because of SHA.


On 20 April 2015 01:14:27 Skip <notifications@github.com> wrote:

> > @@ -0,0 +1,28 @@
> > +{ stdenv, fetchgit, cmake, mesa, libX11, freetype, libjpeg, openal,
> libsndfile
> > +, glew, libXrandr, libXrender
> > +}:
> > +stdenv.mkDerivation rec {
> > +  name = "sfml-git-20110428";
>
> At the time of writing, 2.2 is the most recent tag (which is the version of
> the new default.nix). So I'm not sure what features were implemented after
> that..
>
> Is there a possibility to make it always be HEAD? Or is that not possible
> as we need to update the SHA256 hash all the time?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/NixOS/nixpkgs/pull/7468/files#r28657538